lib/resque/scheduler/configuration.rb

```diff
@@ -8,56 +8,61 @@ def configure
         yield self
       end
 
+      attr_writer :environment
+      def environment
+        @environment ||= ENV
```

Comment:
Why not use `ENV` directly here? So folks can customize the configuration without modifying ENV?

Reply:
@iloveitaly it just makes it easier to write tests to allow this be provided instead of mocking/stubbing.